AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/Keyrings/AwsKms/AwsKmsDiscoveryKeyring.dfy

```diff
@@ -506,6 +530,10 @@ module AwsKmsDiscoveryKeyring {
         && Seq.Last(client.History.Decrypt).output.value.KeyId == Some(keyArn)
     }
 
+    predicate Requires(helper: AwsKmsEdkHelper){
+      true
+    }
+
```

Comment:
What is this doing?

Reply:
Also from https://github.com/aws/aws-cryptographic-material-providers-library/pull/909

Reply:
Needed, or we get `'AwsKmsEncryptedDataKeyDecryptor' does not implement trait predicate 'Action.Requires'`